common/src/main/java/com/wynntils/features/EmeraldPouchHotkeyFeature.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features;
+
+import com.google.common.collect.ImmutableList;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.features.properties.GameplayImpact;
+import com.wynntils.core.features.properties.PerformanceImpact;
+import com.wynntils.core.features.properties.Stability;
+import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.mc.utils.keybinds.KeyHolder;
+import com.wynntils.mc.utils.keybinds.KeyManager;
+import com.wynntils.utils.objects.Pair;
+import com.wynntils.utils.reference.EmeraldSymbols;
+import com.wynntils.wc.utils.WynnItemMatchers;
+import com.wynntils.wc.utils.WynnUtils;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.TranslatableComponent;
+import net.minecraft.network.protocol.game.ServerboundContainerClickPacket;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.inventory.ClickType;
+import net.minecraft.world.item.ItemStack;
+import org.lwjgl.glfw.GLFW;
+
+@FeatureInfo(stability = Stability.STABLE, gameplay = GameplayImpact.MEDIUM, performance = PerformanceImpact.SMALL)
+public class EmeraldPouchHotkeyFeature extends Feature {
+
+    private static final Pattern POUCH_USAGE_PATTERN =
+            Pattern.compile("§6§l(\\d* ?\\d* ?\\d*)" + EmeraldSymbols.E_STRING);
+    private static final Pattern POUCH_CAPACITY_PATTERN =
+            Pattern.compile("\\((\\d+)(" + EmeraldSymbols.BLOCKS + "|" + EmeraldSymbols.LE + "|stx) Total\\)");
```

Comment:
imo this should be moved to `WynnItemMatchers`, and on that note `WynnItemMatchers` should also be for parsing too, which might require a rename

Reply:
Or `EmeraldPouchManager` should be in a dedicated parsing section

Reply:
Imo we should leave WynnItemMatchers to match unique items such as the already included soul points. Items that we'll need to interact with immediately after matching (eg. pouches, amps, powders, potions) will have their own info parsing methods (eg. getPouchCapacity, getPotionHealAmount) that should be near wherever their matchers are.

Maybe we could have all the managers for those under `features/wc/utils/managers`?

Reply:
I'd rather call them parsers versus managers